gulpfile.js

```diff
@@ -267,21 +284,18 @@ gulp.task('dist:scripts', ['tpl:dropdown', 'tpl:file-input', 'tpl:text-field', '
         .pipe(gulp.dest('dist'));
 });
 
-gulp.task('watch', ['build'], function()
-{
-    watcherWithCache('lint', paths.js, ['lint']);
-    watcherWithCache('scss', [paths.scss, 'demo/scss/**/*.scss'], ['scss']);
-    watcherWithCache('demo', paths.demo, ['demo']);
-    watcherWithCache('examples', paths.examples, ['examples']);
-    watcherWithCache('libs', paths.libs, ['libs']);
-    watcherWithCache('tpl:dropdown', 'modules/dropdown/views/*.html', ['tpl:dropdown']);
-    watcherWithCache('tpl:file-input', 'modules/file-input/views/*.html', ['tpl:file-input']);
-    watcherWithCache('tpl:text-field', 'modules/text-field/views/*.html', ['tpl:text-field']);
-    watcherWithCache('tpl:search-filter', 'modules/search-filter/views/*.html', ['tpl:search-filter']);
-    watcherWithCache('tpl:select', 'modules/select/views/*.html', ['tpl:select']);
-    watcherWithCache('tpl:tabs', 'modules/tabs/views/*.html', ['tpl:tabs']);
-    watcherWithCache('tpl:date-picker', 'modules/date-picker/views/*.html', ['tpl:date-picker']);
-    watcherWithCache('tpl:progress', 'modules/progress/views/*.html', ['tpl:progress']);
+gulp.task('connect', function() {
+    connect.server({
+        root: 'build'
+    });
+});
+
+gulp.task('serve', ['build', 'connect'], function () {
+    registerWatchers();
```

Comment:
Why creating the `registerWatchers` function and not just depend on the watch task instead of launching the watchers and depending on build ? (watch already launched the watchers and was depending en build)